contracts/andromeda-merkle-airdrop/src/contract.rs

```diff
@@ -0,0 +1,982 @@
+#[cfg(not(feature = "library"))]
```

Comment:
I think it's good practice that we add a comment referencing the source for a lot of these methods as its not our code. Could you add a comment saying that this contract is based off [this](https://github.com/CosmWasm/cw-tokens/blob/main/contracts/cw20-merkle-airdrop/src/contract.rs).